lib/Targets/AIETargetCDODirect.cpp

```diff
@@ -350,6 +350,10 @@ struct AIEControl {
     //		macro. In future, the same macro will be expanded to allocate
     //		more memory from the user application for resource management.
     XAie_InstDeclare(_devInst, &configPtr);
+
+    // Setting the backend to CDO
+    configPtr.Backend = XAIE_IO_BACKEND_CDO;
```

Comment:
if cdo backend works for vck boards, why do we have AIETargetXAIEV2?

Reply:
It doesn't work on the vck boards. But in our version of aie-rt we added the capability to be able to choose the backend by passing in this backend variable. Otherwise, if you want to switch backends in aie-rt you need to recompile the entire thing with a new backend. So this is necessary when merging the aie-rts.

Reply:
In theory aie-rt can support multiple backends in the same library, it's just not exercised by the build process.